docs/tutorials/installation.md

```
@@ -0,0 +1,42 @@
+# Installation
+
+## Check your version of python
+
+You will need python 3.11 or later. You can check your version of python by
+typing into a terminal:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)